app/src/controllers/pages/selectors.js

```diff
@@ -210,24 +210,26 @@ export const userAssignedSelector = (projectSlug, organizationSlug) => (state) =
   const isAdmin = userRole === ADMINISTRATOR;
   const isManager = organizationRole === MANAGER;
   let isAssignedToTargetOrganization = false;
+  const projectKey = `${organizationSlug}.${projectSlug}`;
+  const project = assignedProjects[projectSlug] || assignedProjects[projectKey];
```

Comment:
Why the `assignedProjects` isn't consistent and may contain entities by slug and by key?
Does it make sense to discuss it with backend?

Reply:
This probably needs to be discussed. But for now, most of the projects won't work without it.

Reply:
Done.